testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{collections::HashSet, fmt::Display};
+
+use failure::_core::fmt::{Error, Formatter};
+use failure::_core::time::Duration;
+
+use crate::experiments::Context;
+use crate::experiments::Experiment;
+use crate::instance::Instance;
+use crate::stats;
+use crate::util::unix_timestamp_now;
+
+pub struct PerformanceBenchmark {
+    instances: Vec<Instance>,
+}
+
+impl PerformanceBenchmark {
+    pub fn new(instances: Vec<Instance>) -> Self {
+        Self { instances }
+    }
+}
+
+impl Experiment for PerformanceBenchmark {
+    fn affected_validators(&self) -> HashSet<String> {
```

Comment:
nit: this could be made default implementation in trait, since it's shared between many experiments

Reply:
Done